src/client/LiveConf.js

```diff
@@ -402,7 +402,7 @@ class LiveConf {
   /*
   * Generate the live recording config as required by QFAB, based on defaults and optional custom settings.
   */
-  generateLiveConf({customSettings}) {
+  generateLiveConf({customSettings, customMetaValues}) {
```

Comment:
What is the difference between customSettings and customMetaValues?

Reply:
Let's replace customSettings then - all we need to do is plumb customMetaValues through generateAudioStreamsConfig() right?  Then we can remove customSettings?